static_frame/core/frame.py

```diff
@@ -3371,19 +3372,15 @@ def isin(self, other) -> 'Frame':
         '''
         Return a same-sized Boolean Frame that shows if the same-positioned element is in the iterable passed to the function.
         '''
-        # cannot use assume_unique because do not know if values are unique
-        v, _ = iterable_to_array(other)
-        # NOTE: is it faster to do this at the block level and return blocks?
-        array = np.isin(self.values, v)
-        array.flags.writeable = False
+        array = isin(self.values, other)
         return self.__class__(array, columns=self._columns, index=self._index)
 
     @doc_inject(class_name='Frame')
     def clip(self,
             lower=None,
             upper=None,
             axis: tp.Optional[int] = None):
-        '''{}
+        '''
```

Comment:
These brackets are significant, they're filled by the `@doc_inject` decorator above.

```suggestion
        '''{}
```